writing/gen-linecount.py

```diff
@@ -0,0 +1,47 @@
+#! /bin/python3
+
+import subprocess
+import json
+import glob
+import shutil
+
+def get_generated_egg():
+    program = subprocess.run(["cargo", "run", "--quiet"], cwd="../dag_in_context", capture_output=True)
+    return len(program.stdout.splitlines())
+
+def get_written_egg():
+    total_lines = 0
+    file_paths = glob.glob("../../**/*.egg", recursive=True)
+    for file_path in file_paths:
+        with open(file_path, 'r') as file:
+            lines = file.readlines()
+            total_lines += len(lines)
+    return total_lines
+
+def get_rust_lines():
+    rust_lines_output = subprocess.run(["scc", "--format", "json", "../../"], capture_output=True)
```

Comment:
We can't use `loc` due to limitations on its output formats (it only outputs ASCII art that would be nearly impossible to parse) and it is no longer maintained, so the author recommended `scc`.

However, there is another line counting tool called `tokei` which also is in Rust and looks like it will do what we want. I will switch this over to tokei